bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.vcxproj

```diff
@@ -23,33 +23,33 @@
     <ProjectGuid>{90174F71-CE49-42A2-8A52-2AA4F529F7D6}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>bakkesmodplaylistrankviewervs</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
     <ProjectName>bakkesmod-playlist-rank-viewer</ProjectName>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
I actually use VS2017 so this breaks my builds :) Im going to try and install VS2019 and hope that fixes it though. Will get back to you